charts/ftl/templates/lease-services.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    {{- include "ftl.labels" . | nindent 4 }}
+  name: {{ include "ftl.fullname" . }}-lease
+  {{- if .Values.lease.service.annotations }}
+  annotations:
+    {{- toYaml .Values.service.annotations | nindent 4 }}
+  {{- end }}
+spec:
+  ports:
+    {{- range .Values.lease.ports }}
```

Comment:
should this be `{{- range .Values.lease.service.ports }}`? and should it be `{{- toYaml .Values.lease.service.annotations | nindent 4 }}` above?

Reply:
yes, it should